package.json

```diff
@@ -25,18 +25,21 @@
     "@emotion/is-prop-valid": "^1.2.0",
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
+    "@ethersproject/address": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
     "@galacticcouncil/apps": "1.7.13",
     "@galacticcouncil/bridge": "0.0.13",
     "@galacticcouncil/math-lbp": "^0.2.1",
     "@galacticcouncil/math-liquidity-mining": "^0.2.0",
-    "@galacticcouncil/math-staking": "^0.2.0",
     "@galacticcouncil/math-stableswap": "0.2.1",
+    "@galacticcouncil/math-staking": "^0.2.0",
     "@galacticcouncil/math-xyk": "^0.2.0",
     "@galacticcouncil/sdk": "^0.7.7",
     "@galacticcouncil/ui": "^1.3.4",
     "@galacticcouncil/xcm": "^0.0.1",
     "@galacticcouncil/xcm-config": "^1.1.3",
     "@lit-labs/react": "^1.1.0",
+    "@metamask/sdk": "^0.11.2",
```

Comment:
do we really need this?

Reply:
not really, but it helps with typescript definitions for the metamask provider

Reply:
not really worth 27 MB dependency in my opinion, pls remove

Reply:
removed Metamask SDK